.env

```diff
@@ -0,0 +1,20 @@
+NODE_DEBUG="@wireapp/*"
+PORT="8081"
+
+APP_BASE="https://app.wire.com/"
+APP_NAME="Webapp"
+BACKEND_REST="https://staging-nginz-https.zinfra.io"
+BACKEND_WS="wss://staging-nginz-ssl.zinfra.io"
+CSP_EXTRA_CONNECT_SRC="http://localhost:32123, ws://localhost:32123, https://*.zinfra.io, https://*.wire.com, https://api.raygun.io, wss://*.zinfra.io, wss://prod-nginz-ssl.wire.com"
+CSP_EXTRA_IMG_SRC="https://*.zinfra.io, https://*.wire.com"
+CSP_EXTRA_SCRIPT_SRC="http://localhost:32123, https://*.zinfra.io, https://*.wire.com, https://api.raygun.io"
+FEATURE_CHECK_CONSENT="true"
+FEATURE_ENABLE_DEBUG="true"
+FEATURE_ENABLE_SSO="true"
+ENFORCE_HTTPS="false"
+URL_ACCOUNT_BASE="https://wire-account-staging.zinfra.io/"
+URL_MOBILE_BASE="https://wire-pwa-staging.zinfra.io/"
+URL_TEAMS_BASE="https://wire-admin-staging.zinfra.io/"
+URL_WEBSITE_BASE="https://wire-website-staging.zinfra.io/"
+#RAYGUN_API_KEY="5hvAMmz8wTXaHBYqu2TFUQ=="
```

Comment:
Should this be here?

Reply:
As a starter, yes. We can remove it once we set up all the environments.